endoreg_db/helpers/test_video_helper.py

```
@@ -0,0 +1,105 @@
+from pathlib import Path
+import random
+import re
+from typing import Optional
+
+from logging import getLogger
+logger = getLogger(__name__)
+
+
+from endoreg_db.utils.video.names import (
+    identify_video_key,
```

Ruff (0.11.9)

`endoreg_db.utils.video.names.identify_video_key` imported but unused

Remove unused import

(F401)